fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/AsyncWebServiceRequestHandlerSpec.groovy

```diff
@@ -42,6 +42,7 @@ class AsyncWebServiceRequestHandlerSpec extends Specification {
         1 * dws.postDruidQuery(rc, _, null, null, groupByQuery) >> { a0, a1, a2, a3, a4 ->
             // Save the success callback
             sc = a1
+            return null
```

Comment:
It looks like it was trying to force something to be a future but was unable to cast it.
```java
GroovyCastException: Cannot cast object 'com.yahoo.bard.webservice.web.handlers.AsyncWebServiceRequestHandler$1@33e01298' with class 'com.yahoo.bard.webservice.web.handlers.AsyncWebServiceRequestHandler$1' to class 'java.util.concurrent.Future'
```